packages/grant-explorer/src/features/discovery/ExploreProjectsPage.tsx

```diff
@@ -3,10 +3,15 @@ import LandingHero from "./LandingHero";
 import { LandingSection } from "./LandingSection";
 import { useCartStorage } from "../../store";
 import { ApplicationStatus, CartProject } from "../api/types";
-import { useMemo } from "react";
+import { useMemo, useState } from "react";
 import { ApplicationSummary } from "common/src/grantsStackDataClientContext";
-import { useApplications } from "./hooks/useApplications";
+import {
+  ApplicationFetchOptions,
+  useApplications,
+} from "./hooks/useApplications";
 import PaginatedProjectsList from "./PaginatedProjectsList";
+import { useDebounce } from "use-debounce";
```

Comment:
I would suggest we use this library because it has a lot of other useful hooks:
https://github.com/streamich/react-use

Reply:
oh nice, will do!

Reply:
the search now works on submit so not relevant anymore